gen/ioctl/generate.sh

```diff
@@ -23,6 +23,9 @@ i686-linux-gnu-gcc main.c list.o -o main.exe $cflags
 x86_64-linux-gnu-gcc -Iinclude -c list.c $cflags
 x86_64-linux-gnu-gcc main.c list.o -o main.exe $cflags
 ./main.exe >> "$out"
+loongarch64-unknown-linux-gnu-gcc -Iinclude -c list.c $cflags
+loongarch64-unknown-linux-gnu-gcc main.c list.o -o main.exe $cflags
```

Comment:
Which distro this script is supposed to get run on? This looks like the Debian/Ubuntu cross toolchain naming but AFAIK the Debian port still hasn't settled yet?